tests/SprykerEcoTest/Zed/Adyen/Business/Facade/ExecutePostSaveHookTest.php

```diff
@@ -0,0 +1,374 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Adyen\Business\Facade;
+
+use Generated\Shared\DataBuilder\AdyenApiResponseBuilder;
+use Generated\Shared\Transfer\AdyenApiMakePaymentResponseTransfer;
+use Generated\Shared\Transfer\AdyenApiResponseTransfer;
+use Generated\Shared\Transfer\CheckoutResponseTransfer;
+use Spryker\Shared\Kernel\Transfer\Exception\NullValueException;
+use SprykerEco\Zed\Adyen\Dependency\Facade\AdyenToAdyenApiFacadeBridge;
+use SprykerEcoTest\Zed\Adyen\Business\BaseSetUpTest;
+
+/**
+ * @group Functional
+ * @group SprykerEco
+ * @group Zed
+ * @group Adyen
+ * @group Business
+ * @group Facade
```

Comment:
try deleting this annotating and running code sniffer again
I expect `ExecutePostSaveHookTest` group to be present